xml/admin_gui_dashboard.xml

```diff
@@ -2199,6 +2199,94 @@ scrape_configs:
+        </para>
+      </note>
+    </sect3>
+    <sect3 xml:id="dash.rbd_qos.qos">
+      <title>Configuration Options</title>
+      <para>
+        Click "Quality of Service +" to expand the configuration options.  A list of all available options will show up.  The units of the configuration options are already shown in the input fields.  In case of any bytes per second (BPS) option, you are free to use values like "1M" or "5G".  They will automatically be converted to "1 MB/s" and "5 GB/s" respectively.
+      </para>
+      <para>
+        By clicking on reset button to the right of each input field, any value set on the *pool* will be removed.  This does not remove configuration values of options configured globally or on an RBD image.
+      </para>
+    </sect3>
+    <sect3 xml:id="dash.rbd_qos.image_create">
+      <title>Creating RBD QoS options with a new RBD image</title>
+      <para>
+        To create an RBD image with RBD QoS options set on that image, click <guimenu>Block</guimenu>, then <guimenu>Images</guimenu> and lastly <guimenu>Create</guimenu>.  By clicking on <guimenu>Advanced</guimenu> the advanced configuration section will expand.  Click on <guimenu>Quality of Service</guimenu> to open all available configuration options.
+      </para>
+    </sect3>
+    <sect3 xml:id="dash.rbd_qos.image_edit">
+      <title>Editing RBD QoS options on existing images</title>
+      <para>
+        To edit RBD QoS options on an existing image, click <guimenu>Block</guimenu>, <guimenu>Images</guimenu>, then click the pools' row in the table and lastly click <guimenu>Edit</guimenu>.  The edit dialog will show up.  By clicking on <guimenu>Advanced</guimenu> the advanced configuration section will expand.  Click on <guimenu>Quality of Service</guimenu> to open all available configuration options.
+      </para>
+    </sect3>
+    <sect3 xml:id="dash.rbd_qos.image_copy">
+     <title>Changing configuration options when copying or cloning images</title>
+     <para>
+       If an RBD image is cloned or copied, the values set on that particular image will, by default, be copied, too.  If you want to change them while copying or cloning, you can do so by specifying the updated configuration values in the copy/clone dialog, just like you would when creating or editing an RBD image.  Doing so will only set (or reset) the values for that RBD image to be copied or cloned. Neither will the source RBD image configuration be changed, nor the global configuration.
+     </para>
+     <para>
+       If you choose to reset the option value on copying/cloning, no value for that option will be set on that image.  This means any value of that option specified for the parent pool will be used, if the parent pools has a value configured. Otherwise, the global default will be used.
```

Comment:
```suggestion
       If you choose to reset the option value on copying/cloning, no value for that option will be set on that image.  This means that any value of that option specified for the parent pool will be used if the parent pool has the value configured. Otherwise, the global default will be used.
```